engine/src/main/java/org/destinationsol/game/planet/PlanetConfigs.java

```diff
@@ -40,26 +38,29 @@ public PlanetConfigs(HullConfigManager hullConfigs, GameColors cols, ItemManager
         medium = new ArrayList<>();
         hard = new ArrayList<>();
 
-        Assets.cacheLists();
+        Set<ResourceUrn> planetJsonConfigs = Assets.getAssetHelper().list(Json.class, "[a-zA-Z0-9]*:planetsConfig");
 
-        Json json = Assets.getJson("core:planetsConfig");
-        JsonValue rootNode = json.getJsonValue();
+        for (ResourceUrn planetConfigJson : planetJsonConfigs) {
+            Assets.cacheLists();
```

Comment:
Actually, we can remove this line.

Yes, I know you did not add this, you just copied it from the existing source - but I looked into the details, and it only caches textures, which are not being queried here.